sass/includes/_record-links.scss

```diff
@@ -0,0 +1,92 @@
+.record-links {
+    &__wrapper {
+        display: flex;
+        align-items: center;
+        flex-direction: column;
+        padding-left:10%;
+        padding-right:10%;
+
+        @media (max-width: $screen__md) {
```

Comment:
Tip: it's best practice (and v helpful) to build from a mobile-first approach. This way, the first styles you add target mobile and then any media queries are for anything above mobile. So you'd do `@media min-width` rather than `@media (max-width ...)`. It's a slightly different way of thinking about it but it makes the styling process much more efficient, mobile browsers only render the relevant styles and it means you don't need to override or re-create the same properties multiple times.